### DISCUSSION TITLE
[@types/react-dom] useFormState: isPending in the return value

### DISCUSSION THREAD

**Discussion by yuto-ono** [DefinitelyTyped/DefinitelyTyped#71161] [Issues with a @types package] [answered]
In @types/react-dom 18.3.1, isPending has been added to the return type of useFormState ( #68974 ). However, isPending is actually undefined in react-dom 18.3.1. react-dom 19rc seems to have it. is it added ahead of time since it exists in RC? I was confused by the seeming discrepancy in the type definitions.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
`useFormState` is a Canary feature.

The published types for experimental and canary are always for the latest `@experimental` or `@canary` release not a specific one.

**Reply by yuto-ono** [reply]
Thank you for your response!
